handlers/upload.go

```diff
@@ -132,7 +132,9 @@ func (s Server) guestEntryPost() http.HandlerFunc {
 			r.Body = http.MaxBytesReader(w, r.Body, int64(*gl.MaxFileBytes))
 		}
 
-		id, err := s.insertFileFromRequest(r, picoshare.NeverExpire, guestLinkID)
+		t := picoshare.ExpirationTime(time.Now().Add(gl.FileLifetime.Duration()))
```

Comment:
This feels like it's breaking the encapsulation a little bit in that the caller has to reconstruct a `picoshare.ExpirationTime` and peek into the `FileLifetime`'s `Duration` object.

Could we add a function to `FileLifetime` so that the caller can do this:

```go
expirationTime := gl.FileLifetime.ExpirationFromTime(time.Now())
```